tests/__fixtures__/app/Models/ExternalPost.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
We're only using this model in the `ResourceTest` file. To keep things self-contained, let's move this to the bottom of that file.